### PR TITLE
Feature/wv 2428

### PR DESCRIPTION
**Pull request by arthurlorenz**

### CODE REVIEW

File: modules/contentpool_custom_elements/contentpool_custom_elements.libraries.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  css: {}
+  js:
+    'js/load-vue-components.js': {}
+  dependencies:
+    - contentpool_custom_elements/vue
+
+vue:
+  version: 2.5.16
+  js:
+    'https://cdn.jsdelivr.net/npm/vue@2.5.16/dist/vue.js':
```
Comment (fago) [inline modules/contentpool_custom_elements/contentpool_custom_elements.libraries.yml]: please add "vue" module for that and depend on its library. that's what we already do on the client side, see https://github.com/drunomics/contentpool-client/pull/15

**Review by fago — CHANGES_REQUESTED** [review]

File: composer.json [hunk 2]
```diff
@@ -34,6 +34,9 @@
         "#1356276-417: Allow profiles to provide a base/parent profile and load them in the correct order": "https://www.drupal.org/files/issues/drupal-n1356276-417-d8.5.%2A.patch",
         "#2893029: StringTranslationTrait consumers cannot be reliably serialized without DependencySerializationTrait": "https://www.drupal.org/files/issues/2893029-full-fix-test-from-3.patch"
       },
+      "drupal/custom_elements": {
```
Comment (fago) [inline composer.json:37]: well, first of drupal/custom_elements needs to be added as dependency?

File: modules/contentpool_custom_elements/contentpool_custom_elements.info.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+type: module
+name: Contentpool Custom elements
```
Comment (fago) [inline modules/contentpool_custom_elements/contentpool_custom_elements.info.yml]: Contentpool custom elements

File: modules/contentpool_custom_elements/contentpool_custom_elements.info.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+description: 'Implements custom elements for Contentpool'
```
Comment (fago) [inline modules/contentpool_custom_elements/contentpool_custom_elements.info.yml]: Adds default-styling for the custom-elements of thunder-paragraphs.

File: modules/contentpool_custom_elements/contentpool_custom_elements.info.yml [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+package: Contentpool
+core: '8.x'
```
Comment (fago) [inline modules/contentpool_custom_elements/contentpool_custom_elements.info.yml]: please add configuration to the contentpool which renders the article paragraph field as custom-elements, so this is actually applied.
next step is using this during replication, but that's another issue then.

**Review by fago — CHANGES_REQUESTED** [review]

File: modules/contentpool_custom_elements/contentpool_custom_elements.info.yml (hunk 5 above)
Comment (arthurlorenz) [reply modules/contentpool_custom_elements/contentpool_custom_elements.info.yml]: i don't really know what you mean, what should be applied?

File: modules/contentpool_custom_elements/contentpool_custom_elements.libraries.yml [hunk 6]
```diff
@@ -8,21 +8,21 @@ main:
 
 vue-lupus-paragraphs:
   js:
-    'https://drunomics.github.io/vue-lupus-paragraph-text/dist/main.js':
+    'http://rawgit.com/drunomics/vue-lupus-paragraph-text/0.1.0/dist/main.js':
```
Comment (fago) [inline modules/contentpool_custom_elements/contentpool_custom_elements.libraries.yml]: should use the https version.

**Review by fago — CHANGES_REQUESTED** [review]

**Review by fago — APPROVED** [review]

**Pull request merged**